db/seeds.rb

```diff
@@ -47,9 +47,9 @@
 ronald4 = User.find(4)
 
 # tenants
-Tenant.create!(
+bridge_builders = Tenant.create!(
   location:              "East Timor represent",
-  organization:          "Bridge Builders off the island."
+  organization:          "bridge-builders"
```

Comment:
I agree that this branch doesn't involve that, but we should still revert the seed file in this case to its former state to better reflect reality.